javascript/react/src/hooks/downloads.ts

```diff
@@ -56,7 +57,12 @@ export function useDownloadZip(
 		}
 
 		if (tables.length) {
-			const files = tables
+			const map = new Map<string, number>()
+			const renamedTables = tables.map(t => ({
+				...t,
+				name: renameDuplicatedFileName(map, t.name || t.id || 'table.csv'),
```

Comment:
will we run into an issue if there are multiple files without names?

Reply:
I see, the renameDupliactedFileName will resolve the name for us

Reply:
I checked, and we might. So far, when a csv file is uploaded, the id of the tableContainer object created for that file is the file name. I tried changing it to be a random string, but it broke the table select, as it showed the id of the selected file and not its name as it currently does.